### DISCUSSION TITLE
Sort by Most Recent

### DISCUSSION THREAD

**Discussion by mrevjd** [dailydotdev/daily#392] [Feature Requests]
There used to be an option where you could sort by the most recent. In the new UI it seems it is no longer there.

**Comment by mrevjd** [comment]
https://github.com/dailydotdev/daily/discussions/395#discussion-3765448

**Reply by idoshamun** [reply]
Let's keep the discussion there :)

**Reply by nimrodkra** [reply]
The change is live: https://app.daily.dev/posts/e-dQ6UyIA
